Src/WindowsServer/WindowsServer.Shared.Tests/AzureRoleEnvironmentTelemetryInitializerTest.cs

```diff
@@ -1,50 +1,32 @@
 ﻿namespace Microsoft.ApplicationInsights.WindowsServer
-{
+{    
+    using System;
+#if NET45
+    using System.Diagnostics.Tracing;
+#endif
     using System.Globalization;
+    using System.IO;
+    using System.Linq;    
     using Microsoft.ApplicationInsights.DataContracts;
     using Microsoft.ApplicationInsights.Extensibility.Implementation;
-
     using Microsoft.ApplicationInsights.WindowsServer.Azure;
     using Microsoft.ApplicationInsights.WindowsServer.Azure.Emulation;
     using Microsoft.ApplicationInsights.WindowsServer.Implementation;
-    
+#if NET40
+    using Microsoft.Diagnostics.Tracing;
+#endif
     using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Web.TestFramework;
     using Assert = Xunit.Assert;
 
     [TestClass]
     public class AzureRoleEnvironmentTelemetryInitializerTest
-    {
-        [TestMethod]
-        public void AzureRoleEnvironmentTelemetryInitializerSetsTelemetryContextPropertiesWhenRoleEnvironmentIsAvailable()
-        {
-            var telemetryItem = new EventTelemetry();
-            var initializer = new AzureRoleEnvironmentTelemetryInitializer();
-            AzureRoleEnvironmentContextReader.BaseDirectory = ServiceRuntimeHelper.TestWithServiceRuntimePath;
-            AzureRoleEnvironmentContextReader.Instance = null;
-
-            ServiceRuntimeHelper.IsAvailable = true;
-            initializer.Initialize(telemetryItem);
-
-            string expectedRoleInstanceName = string.Format(
-                                    CultureInfo.InvariantCulture,
-                                    TestRoleInstance.IdFormat,
-                                    ServiceRuntimeHelper.RoleName,
-                                    ServiceRuntimeHelper.RoleInstanceOrdinal);
-
-            Assert.Equal(ServiceRuntimeHelper.RoleName, telemetryItem.Context.Cloud.RoleName);
-            Assert.Equal(expectedRoleInstanceName, telemetryItem.Context.Cloud.RoleInstance);
-            Assert.Equal(expectedRoleInstanceName, telemetryItem.Context.GetInternalContext().NodeName);
-        }
-
+    {        
         [TestMethod]
         public void AzureRoleEnvironmentTelemetryInitializerDoesNotOverrideRoleName()
         {
-            var telemetryItem = new EventTelemetry();
+            var telemetryItem = new EventTelemetry();            
```

Comment:
nitpick: seems like there are a bunch of spaces on the end of all these lines as the only change, I think VS might have an option to strip trailing whitespace but I don't recall.